### PR TITLE
Mandd/relap5interface mods

### LINKED ISSUE #208: RELAP5 handling of output files
--------
Issue Description
--------
##### What did you expect to see happen?
The actual method that generates .csv files from .o files needs better handling of the RELAP5 output in order to avoid that the .csv file contains string

##### What did you see instead?
the .csv might contain string which causes RAVEN to crash

##### Please attach the input file(s) that generate this error.  The simpler the input, the faster we can find the issue.

----------------
For Change Control Board: Issue Review
----------------
This review should occur before any development is performed as a response to this issue.
- [x] 1. Is it tagged with a type: defect or improvement? improvement
- [x] 2. Is it tagged with a priority: critical, normal or minor? normal
- [x] 3. If it will impact requirements or requirements tests, is it tagged with requirements? NO
- [x] 4. If it is a defect, can it cause wrong results for users? If so an email needs to be sent to the users. NO
- [x] 5. Is a rationale provided? (Such as explaining why the improvement is needed or why current code is wrong.) Yes

-------
For Change Control Board: Issue Closure
-------
This review should occur when the issue is imminently going to be closed.
- [x] 1. If the issue is a defect, is the defect fixed? not a defect
- [x] 2. If the issue is a defect, is the defect tested for in the regression test system?  (If not explain why not.) NA
- [x] 3. If the issue can impact users, has an email to the users group been written (the email should specify if the defect impacts stable or master)? NA
- [x] 4. If the issue is a defect, does it impact the latest stable branch? If yes, is there any issue tagged with stable (create if needed)? NA
- [x] 5. If the issue is being closed without a merge request, has an explanation of why it is being closed been provided? NA

### PR DESCRIPTION
**Pull request by mandd**
--------
Pull Request Description
--------
##### What issue does this change request address? (Use "#" before the issue to link it, i.e., #42.)
Closes #208

##### What are the significant changes in functionality due to this change request?
Better handling of the RELAP5 output files

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [x] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [x] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [x] 8. If an analytic test is changed/added is the the analytic documentation updated/added?

### CODE REVIEW

File: framework/CodeInterfaces/RELAP5/relapdata.py [hunk 1]
```diff
@@ -157,6 +153,20 @@ def readMinorBlock(self,lines,i):
           flagg2=1
     return minorDict
 
+  def checkLine(self,list):
+    """
+      Method that check the content of a list (i.e., a line); a list must contain only numbers
```
Comment (alfoa) [inline framework/CodeInterfaces/RELAP5/relapdata.py]: ```that checks```

File: framework/CodeInterfaces/RELAP5/relapdata.py [hunk 2]
```diff
@@ -157,6 +153,20 @@ def readMinorBlock(self,lines,i):
...
+      @ In, list, list, list that contained values located in a single line
```
Comment (alfoa) [inline framework/CodeInterfaces/RELAP5/relapdata.py]: ```list``` is a built-in datatype in Python. Even if you can assign a variable name to that, it is generally discouraged. Can you change the variable name to something else? (e.g. ```lineList```)

File: framework/CodeInterfaces/RELAP5/relapdata.py (hunk 1 above)
Comment (mandd) [inline framework/CodeInterfaces/RELAP5/relapdata.py]: fixed

File: framework/CodeInterfaces/RELAP5/relapdata.py (hunk 2 above)
Comment (mandd) [inline framework/CodeInterfaces/RELAP5/relapdata.py]: this is actually a good point: Fixed

**Comment by alfoa** [comment]
Diego,
I added my commits... Can you review them so I can review the rest?

**Comment by mandd** [comment]
I approved the modifications performed by alfoa

**Comment by alfoa** [comment]
----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [ ] 1. Review all computer code. ***done by me and @mandd***
- [ ] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts). ***ok***
- [ ] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details. ***ok***
- [ ] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass. ***ok***
- [ ] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True. ***n/a***
- [ ] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync. ***n/a***
- [ ] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done. ***#208***
- [ ] 8. If an analytic test is changed/added is the the analytic documentation updated/added? ***n/a***

**Comment by alfoa** [comment]
Checklist passed...

Merging...

**Pull request merged**